packages/acquirer-frontend/src/pages/ApprovedMerchantRecords/ApprovedMerchantRecords.tsx

```diff
@@ -33,6 +33,8 @@ import {
   TableSkeleton,
 } from '@/components/ui'
 import { FormInput } from '@/components/form'
+import { useUsers } from '@/api/hooks/users'
+import { FormSelect } from '@/components/form/index'
```

Comment:
`<FormSelect />` can be imported from '@/components/form' just like `<FormInput />`.

eg: import { FormInput, FormSelect } from '@/components/form'

Reply:
sry... it was autocomplete from lsp... i will change it